### PR TITLE
feat: limit sign-in password length to 72

### PR DESCRIPTION
**Pull request by hf**
Anything over 72 is not acceptable.

### CODE REVIEW

File: apps/studio/components/interfaces/SignIn/SignInForm.tsx [hunk 1]
```diff
@@ -14,7 +14,9 @@ import { Button, Form, Input } from 'ui'
 
 const signInSchema = object({
   email: string().email('Must be a valid email').required('Email is required'),
-  password: string().required('Password is required'),
+  password: string()
+    .required('Password is required')
+    .max(72, 'Password must be at most 72 characters'),
```
Comment (alaister) [inline apps/studio/components/interfaces/SignIn/SignInForm.tsx:19]: @hf Should we be validating here on sign in? Will gotrue choke if it receives a string longer than 72 characters?

Comment (alaister) [inline apps/studio/components/interfaces/SignIn/SignInForm.tsx:19]: @hf Should we be validating here on sign in? Will gotrue choke if it receives a string longer than 72 characters?

Comment (joshenlim) [inline apps/studio/components/interfaces/SignIn/SignInForm.tsx:19]:
```suggestion
    .max(72, 'Password cannot exceed 72 characters'),
```

File: apps/studio/lib/schemas.ts [hunk 2]
```diff
@@ -5,5 +5,10 @@ import YupPassword from 'yup-password'
 YupPassword(yup)
 
 export const passwordSchema = yup.object({
-  password: yup.string().password().required().label('Password'),
+  password: yup
+    .string()
+    .password()
+    .required()
+    .max(72, 'Password must be at most 72 characters')
```
Comment (joshenlim) [inline apps/studio/lib/schemas.ts:12]:
```suggestion
    .max(72, 'Password cannot exceed 72 characters')
```

File: apps/studio/components/interfaces/SignIn/SignInForm.tsx (hunk 1 above)
Comment (hf) [reply apps/studio/components/interfaces/SignIn/SignInForm.tsx]: Hmm good point this will block people from signing in.

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**